### PR TITLE
changes in README

### PR DESCRIPTION
**Pull request by mbiarnes**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -497,6 +497,9 @@ Any dependency used in any KIE project must fulfill these hard requirements:
     * Forbidden: no license, GPL, AGPL, proprietary license, field of use restrictions ("this software shall be used for good, not evil"), ...
 
         * Even test scope dependencies cannot use these licenses.
+        
+    * To check the ALS compatibily lcense please visit these links:[Similarity in terms to the Apache License 2.0](http://www.apache.org/legal/resolved.html#category-a)&nbsp;
```
Comment (manstis) [inline README.md]: Typo! "lcense"

**Comment by mbiarnes** [comment]
@manstis typo fixed

**Comment by manstis** [comment]
+1 to merge.

**Comment by csadilek** [comment]
master: 2992bbe154deb3c3b714cbf5c28791fbf982bf63

**Pull request closed without merging**